### PR TITLE
feat/cloud-74: Add provenance to our public Helm charts

### PR DESCRIPTION
**Pull request by awsbot-labs**
Encoded GPG key, and updates to the .travis.yml and
package.sh for signing Helm Charts.

### CODE REVIEW

File: package.sh [hunk 1]
```diff
@@ -12,7 +12,7 @@ echo "Packaging charts..."
 # Iterate over all charts are package them
 for dir in ${TRAVIS_BUILD_DIR}/charts/*; do
     helm dep update $dir
-    helm package -d ${TRAVIS_BUILD_DIR}/build $dir
+    helm package --sign --key 'Landoop Helm Charts' --keyring key.gpg -d ${TRAVIS_BUILD_DIR}/build $dir
```
Comment (Antwnis) [inline package.sh]: Lenses.io Helm Charts

**Comment by spirosoik** [comment]
@awsbot-labs squash your commit to one.

**Comment by spirosoik** [comment]
@awsbot-labs as we are here let's add a `Makefile` which triggers the necessary scripts. Let's consolidate it everywhere.

**Comment by spirosoik** [comment]
@awsbot-labs rebase, squash the commits to one and remove from commit `feat/cloud-74`

**Comment by spirosoik** [comment]
let's just create the Makefile too as we discussed.

**Pull request merged**